### ISSUE TITLE
optional dependencies and Server loadings

### ISSUE THREAD

**Issue by fiorenzino** [FroMage/redpipe#24]
we could evaluate in config file to initialize in Server class:
- config file

I'm thinking about:
- for "cookies and session" (if we have "cookies": true in config)
- db client (if we have url or driver_class or db_driver in config)
- swagger ("swagger": true in config )

What do you think about?

**Comment by FroMage** [comment]
I agree, but I'd keep the current defaults, so the config would be useful to disable them.

**Comment by rdruss** [comment]
Should it continue to be referenced as Swagger or OpenAPI?
